### PR TITLE
[nl] Added some missing translations in nl.json

### PR DESCRIPTION
**Pull request by Loots-it**
Added some translations for the jetstream app scaffolding

### CODE REVIEW

File: json/nl.json [hunk 1]
```diff
@@ -1,46 +1,61 @@
...
-    "Forgot your password?": "Forgot your password?",
-    "Forgot Your Password?": "Wachtwoord vergeten?",
+    "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Wachtwoord vergeten? Geen probleem. Geef hier uw e-mailadres in en we sturen u een link via mail waarmee u een nieuw wachtwoord kan instellen.",
+    "Forgot your password?": "Wachtwoord vergeten?",
     "Go Home": "Terug naar de voorpagina",
     "Hello!": "Hallo!",
     "hi": "hallo",
+    "If necessary, you may logout of all of your other browser sessions across all of your devices. If you feel your account has been compromised, you should also update your password.": "Indien nodig kunt u zich afmelden op alle andere browser sessies op al uw apparaten. Als u denkt dat uw account is gehackt, kunt u ook best uw wachtwoord aanpassen.",
```
Comment (caouecs) [inline json/nl.json:24]: This tag was updated yesterday : *"If necessary, you may logout of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password"*

**Review by caouecs — CHANGES_REQUESTED** [review]
Dankjewel, but just a problem on an updated tag

File: json/nl.json (hunk 1 above)
Comment (Loots-it) [reply json/nl.json]: Ok thanks! I fixed it in the next 2 commits (I'm sorry for the typo).

File: json/nl.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
     "If you did not create an account, no further action is required.": "Als je geen account hebt aangemaakt hoef je verder niets te doen.",
     "If you did not receive the email": "Als je de e-mail niet hebt ontvangen:",
     "If you did not request a password reset, no further action is required.": "Als je geen wachtwoordherstel hebt aangevraagd, hoef je verder niets te doen.",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Als je problemen hebt met de \":actionText\" knop, kopieer en plak de URL hieronder\nin je webbrowser:",
     "Invalid signature.": "Ongeldige handtekening.",
     "Login": "Inloggen",
-    "Logout Other Browser Sessions": "Afmelden bij alle sessies",
+    "Logout Other Browser Sessions": "Afmelden bij alle sessies",git
```
Comment (caouecs) [inline json/nl.json]: oups ;)

**Review by caouecs — CHANGES_REQUESTED** [review]
Just a typo 

thank you

**Comment by Loots-it** [comment]
Not sure if I still have to do something?

**Comment by caouecs** [comment]
Dankjewel

**Pull request merged**